src/back/glsl/mod.rs

```diff
@@ -194,9 +194,11 @@ struct FunctionCtx<'a> {
     info: &'a FunctionInfo,
     /// The expression arena of the current function being written
     expressions: &'a Arena<Expression>,
+    /// Map of expressions that have associated variable names
+    named_expressions: &'a crate::NamedExpressions,
 }
 
-impl<'a> FunctionCtx<'a> {
+impl<'a> FunctionCtx<'_> {
```

Comment:
do we still use the `<'a>` thing? if not, we should just write `impl FunctionCtx<'_>`